netdata-installer.sh

```diff
@@ -628,6 +634,80 @@ bundle_libwebsockets
 
 # -----------------------------------------------------------------------------
 
+build_judy() {
+  local env_cmd=''
+
+  if [ -z "${DONT_SCRUB_CFLAGS_EVEN_THOUGH_IT_MAY_BREAK_THINGS}" ]; then
+    env_cmd="env CFLAGS= CXXFLAGS= LDFLAGS="
+  fi
+
+  pushd "${1}" > /dev/null || exit 1
+  run ${env_cmd} libtoolize --force --copy
+  run ${env_cmd} aclocal
+  run ${env_cmd} autoheader
+  run ${env_cmd} automake --add-missing --force --copy --include-deps
+  run ${env_cmd} autoconf
+  run ${env_cmd} ./configure
+  run ${env_cmd} make -C src
+  run ${env_cmd} ar -r src/libJudy.a src/Judy*/*.o
```

Comment:
Any step of those failing should stop the rest from being called. In my system `libtoolize` is not available for instance.

Reply:
Excellent point.

Reply:
And actually, I think we're missing this kind of behavior elsewhere in our bundling code. I'll look into that later though.